x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/downsample/DownsampleTask.java

```diff
@@ -44,11 +42,6 @@ public DownsampleConfig config() {
         return config;
     }
 
-    @Override
-    public Status getStatus() {
-        return status;
-    }
-
     @Override
     public void onCancelled() {
         // TODO(talevy): make things cancellable
```

Comment:
This is not relevant to this PR, but I think DownsampleTask is cancellable, so this comment should be removed